examples/json_serialization/specification.yml

```diff
@@ -16,6 +16,6 @@ holder_disclosed_claims:
   family_name: true
   address: true
 
-key_binding: True
+key_binding: False
```

Comment:
We might want to change the way the python lib does key_binding with json serialization_format, if this PR goes in and sticks, but for now I just flipped this to false.

Reply:
noted